src/components/ImageView/index.native.js

```diff
@@ -86,14 +57,56 @@ class ImageView extends PureComponent {
      * @param {GestureState} gestureState
      * @returns {Boolean}
      */
-    updatePanResponderTouches(e, gestureState) {
+    const updatePanResponderTouches = (e, gestureState) => {
         if (_.isNumber(gestureState.numberActiveTouches)) {
-            this.amountOfTouches = gestureState.numberActiveTouches;
+            numberOfTouches.current = gestureState.numberActiveTouches;
         }
 
         // We don't need to set the panResponder since all we care about is checking the gestureState, so return false
         return false;
-    }
+    };
+
+    // PanResponder used to capture how many touches are active on the attachment image
+    const panResponder = React.useRef(
```

Comment:
Can we move it up where all `useRef` defined?

Reply:
Not really, it has to be defined after  `updatePanResponderTouches` that it uses